sci-libs/openlibm/openlibm-0.7.2.ebuild

```diff
@@ -1,23 +1,34 @@
-# Copyright 1999-2017 Gentoo Foundation
+# Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
 
-inherit eutils toolchain-funcs fortran-2
+inherit toolchain-funcs
 
 DESCRIPTION="High quality system independent, open source libm"
 HOMEPAGE="https://github.com/JuliaLang/openlibm"
-SRC_URI="https://github.com/JuliaLang/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+SRC_URI="https://github.com/JuliaMath/openlibm/archive/v${PV}.tar.gz -> ${P}.tar.gz"
 
 LICENSE="public-domain MIT ISC BSD-2 LGPL-2.1+"
 SLOT="0/${PV}.0"
```

Comment:
Huh. Weird subslot format, but OK.

Reply:
Not sure why it is present.
But because it was present already, I didn't want to remove in case some esoteric julia thing breaks.

Reply:
A subslot is quite common to need to force a rebuild of consumers, I just don't get why they bother with the .0. But it's alright.